src/lexer.coffee

```diff
@@ -538,6 +538,8 @@ exports.Lexer = class Lexer
     if size > @indent
       if noNewlines
         @indebt = size - @indent unless backslash
+        if @indebt
```

Comment:
It might be nice to additionally rename `@indebt` to something more descriptive like `@continuationLineAdditionalIndent`?

Reply:
Sure, `indebt` is a pretty bad name (too close to `indent`).

Reply:
Ok renamed `indebt` -> `continuationLineAdditionalIndent`